### ISSUE TITLE
Reading a json or html or txt file into R-Instat

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#7714]
Here is the file
[wikivital_mathematics.zip](https://github.com/africanmathsinitiative/R-Instat/files/9240241/wikivital_mathematics.zip)

It is a json file, which should import - but it seems not to be a data frame.  So File > Import Data Frame doesn't work.  Is there something that can be done, either with this dataset - which seems interesting - or with the import dialogue, for data of this type?

This is not a very important topic, so can wait until staff have time

**Comment by rdstern** [comment]
The data are available on the [Minor Planet Centre](https://minorplanetcenter.net/data).  

here is the location:
![image](https://user-images.githubusercontent.com/11226469/186858519-c4ab78f8-0f4c-474e-b0a8-702be88cf3f6.png)

There are 3 alternative files and each one gives some problems

a) The txt file imports (seemingly) easily.  But there are additional fields that start after many rows - as indicated in the descriptive file.  If we could have a fixed format - as there is in Excel, then we could do better.  Perhaps we should recommend going via Excel for this situation?
b) I don't know whether the json file can be imported?
c) The description file is clear and then includes the first 5000 records.  It is an html file which I don't know how to import.

Still not particularly urgent.  I assume it is a @Patowhiz challenge?

**Comment by Patowhiz** [comment]
@rdstern we do support importing html and json files in the normal file import dialog. Rio supports html tables by default, please note it must be a html table. To import complicated htmls we have to implement web scrapping.

@anastasia-mbithe could you quickly have a look at whether the attached file imports correctly. If it does kindly outline for us the steps.

**Comment by anastasia-mbithe** [comment]
@Patowhiz, I've been looking into this, for now, there's an error as @rdstern had noted that the dataset is not a data frame. It seems the Rio package doesn't support importing JSON files directly, we have to use jsonlite::fromJSON() and its imported to R as a list,  in our case with the dataset proposed, a large list. 

> The data are available on the [Minor Planet Centre](https://minorplanetcenter.net/data).

I tried to convert the data from this file/site to a data frame and I got 1 observation with 3139645 variables, which would appear so oddly in R-Instat.
